service/controller/clusterapi/v28/credential/credential.go

```diff
@@ -2,16 +2,22 @@ package credential
 
 import (
 	"github.com/giantswarm/microerror"
-	v1 "k8s.io/api/core/v1"
-	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 
-	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v28/legacykey"
+	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v28/key"
 )
 
 const (
-	// awsOperatorArnKey is the key in the Secret under which the ARN for the aws-operator role is held.
-	awsOperatorArnKey = "aws.awsoperator.arn"
+	// AWSOperatorArnKey is the key in the Secret under which the ARN for the
+	// aws-operator role is held.
+	AWSOperatorArnKey = "aws.awsoperator.arn"
```

Comment:
I think this is more useful from a user point of view when this is available by default via godoc.